assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
let's take a following example of scala code:
```
<a data-focuses="@if(something) { some-id } else { some-other-id }">...</a>
```
This will print data-focuses with spaces inside. Good luck trimming it.